_dprhtml/js/search.js

```diff
@@ -900,7 +900,7 @@ function createTables(xmlDoc,hiert)
 
 
                   // paragraph
-                  finalout += ', para. ' + (tmp + 1) + ' <span class="abut obut" onmouseup="DPR1_send_mod.openPlace(' + `${sectionId}, ` + '[\''+nikaya+'\',' + (book - 1) + ',' + sx + ',' + sy + ',' + sz + ',' + s + ',' + se + ',\''+hiert+'\'],' + (tmp+1) + ',\'' + sraout + '\',DPR1_send_mod.eventSend(event))">&rArr;</span></span></p><p>' + DPR1_format_mod.preparepali(sectionId,postpara,1)[0] + '</p><hr></div>';
+                  finalout += ', para. ' + (tmp + 1) + ' <span class="abut obut" onmouseup="DPR1_send_mod.openPlace(' + `${sectionId}, ` + '[\''+nikaya+'\',' + (book - 1) + ',' + sx + ',' + sy + ',' + sz + ',' + s + ',' + se + ',\''+hiert+'\'],' + (tmp+1) + ',\'' + sraout + '\',DPR1_send_mod.eventSend(event,1))">&rArr;</span></span></p><p>' + DPR1_format_mod.preparepali(sectionId,postpara,3)[0] + '</p><hr></div>';

```

Comment:
Same feedback as eventsend below

Reply:
Ok. Or shall I not use the event handler here, just give the value "search" for the parameter "add" ?